src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -589,7 +560,7 @@ public void It_publishes_correctly_in_PublishRelease_evaluation_despite_option_f
         public void It_publishes_on_release_if_PublishRelease_property_set_in_csproj()
         {
             var helloWorldAsset = _testAssetsManager
-               .CopyTestAsset("HelloWorld", "PublishReleaseHelloWorldCsProj")
+               .CopyTestAsset("HelloWorld")
                .WithSource()
                .WithTargetFramework(ToolsetInfo.CurrentTargetFramework)
```

Comment:
You might want to target "net7.0" here, if you want to test that PublishRelease in the project file has an effect, since if you target .NET 8 then it will be set to true anyway and the value in the project doesn't really change anything.